File: src/main/java/io/vproxy/vfx/util/imagewrapper/CanvasBox.java
package io.vproxy.vfx.util.imagewrapper;

public interface CanvasBox {
    void setPaint(int rgb);

    void fillOval(int x, int y, int w, int h);

    void setFont(String name, boolean bold, int size);

    void drawString(String s, int x, int y);

    void flush();
}

File: src-ts/src/util/imagewrapper/CanvasBox.ts
export interface CanvasBox {
  setPaint(rgb: number): void;

  fillOval(x: number, y: number, w: number, h: number): void;

  setFont(name: string, bold: boolean, size: number): void;

  drawString(s: string, x: number, y: number): void;

  flush(): void;
}
